Makefile

```diff
@@ -208,6 +212,49 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### gen_csv: generate the csv for a newer version
+gen_csv:
+	operator-sdk generate csv --csv-version 1.0.1 --from-version 1.0.0 --make-manifests=false --update-crds
+	
+	# filter the deployments so that only the valid deployment is available. See: https://github.com/eclipse/che/issues/17010
+	cat ./deploy/olm-catalog/che-workspace-operator/1.0.1/che-workspace-operator.v1.0.1.clusterserviceversion.yaml | \
+	yq -Y \
+	'.spec.install.spec.deployments[] |= select( .spec.selector.matchLabels.app? and .spec.selector.matchLabels.app=="che-workspace-controller")' | \
+	tee ./deploy/olm-catalog/che-workspace-operator/1.0.1/che-workspace-operator.v1.0.1.clusterserviceversion.yaml >>/dev/null
+
+### push_olm_registry: push the olm application registry to quay.io. Environment variables QUAY_USERNAME, QUAY_PASSWORD, QUAY_NAMESPACE must be set
+push_olm_registry:
+	./olm/push-olm-files-to-quay.sh || true
+
+### olm_build_catalog: build the catalog that will be deployed on the cluster
+olm_build_catalog:
+	docker build -t $(CATALOG_IMG) -f ./deploy/olm-catalog/Dockerfile ./deploy/olm-catalog
+	docker push $(CATALOG_IMG)
+
+### olm_start: use the catalogsource and operatorsource to make the operator be available on the marketplace. Application registry must already be pushed to quay and crd image built and pushed.
+olm_start:
```

Comment:
The registry image I'm getting `quay.io/che-incubator/che-workspace-crd-plugin-registry:7.1.0-offline`, which seems out of date compared to the current default.

Reply:
I think https://github.com/che-incubator/che-workspace-operator/pull/83#discussion_r430075364 might be the issue if I'm understanding correctly